README.md

```diff
@@ -7,24 +7,19 @@
 
 <!-- badges: end -->
 
-Shiny is a new package from RStudio that makes it incredibly easy to build interactive web applications with R.
-
-For an introduction and examples, visit the [Shiny Dev Center](https://shiny.rstudio.com/).
-
-If you have general questions about using Shiny, please use the [RStudio Community website](https://community.rstudio.com). For bug reports, please use the [issue tracker](https://github.com/rstudio/shiny/issues).
+Easily build rich and productive interactive web apps in R &mdash; no HTML/CSS/JavaScript required.
 
 ## Features
 
-* Build useful web applications with only a few lines of code&mdash;no JavaScript required.
-* Shiny applications are automatically "live" in the same way that spreadsheets are live. Outputs change instantly as users modify inputs, without requiring a reload of the browser.
-* Shiny user interfaces can be built entirely using R, or can be written directly in HTML, CSS, and JavaScript for more flexibility.
-* Works in any R environment (Console R, Rgui for Windows or Mac, ESS, StatET, RStudio, etc.).
-* Attractive default UI theme based on [Bootstrap](https://getbootstrap.com/).
-* A highly customizable slider widget with built-in support for animation.
-* Prebuilt output widgets for displaying plots, tables, and printed output of R objects.
-* Fast bidirectional communication between the web browser and R using the [httpuv](https://github.com/rstudio/httpuv) package.
-* Uses a [reactive](https://en.wikipedia.org/wiki/Reactive_programming) programming model that eliminates messy event handling code, so you can focus on the code that really matters.
-* Develop and redistribute your own Shiny widgets that other developers can easily drop into their own applications (coming soon!).
+* An intuitive and extensible [reactive](https://en.wikipedia.org/wiki/Reactive_programming) programming model which makes it easy to transform existing R code into a "live app" where outputs automatically react to new user input.
+  * This programming model naturally leads to more efficient and simpler code then the alternative (i.e., event-based programming) because, when Shiny receives new user input, it intelligently finds the minimal amount of work necessary to update output(s).
```

Comment:
I think you could shrink this bullet and include it in the previous. Something like: "Compared to event-based programming, reactivity makes it much easier for Shiny to do the minimum amount of work when an input changes.